<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-01-05T14:11:19 IST" name="Default suite" finished-at="2022-01-05T14:11:39 IST" duration-ms="20619">
    <groups>
    </groups>
    <test started-at="2022-01-05T14:11:19 IST" name="Default test" finished-at="2022-01-05T14:11:39 IST" duration-ms="20619">
      <class name="TestCases.TC_LoginPage">
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:TestCases.TC_LoginPage@741a8937]" started-at="2022-01-05T14:11:19 IST" name="beforeClass" finished-at="2022-01-05T14:11:19 IST" duration-ms="243" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:TestCases.TC_LoginPage@741a8937]" started-at="2022-01-05T14:11:19 IST" name="beforeMethod" finished-at="2022-01-05T14:11:25 IST" duration-ms="6070" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="textLogin_01()[pri:0, instance:TestCases.TC_LoginPage@741a8937]" started-at="2022-01-05T14:11:25 IST" name="textLogin_01" finished-at="2022-01-05T14:11:33 IST" duration-ms="7311" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- textLogin_01 -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:TestCases.TC_LoginPage@741a8937]" started-at="2022-01-05T14:11:33 IST" name="afterMethod" finished-at="2022-01-05T14:11:33 IST" duration-ms="630" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:TestCases.TC_LoginPage@741a8937]" started-at="2022-01-05T14:11:33 IST" name="beforeMethod" finished-at="2022-01-05T14:11:37 IST" duration-ms="4259" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="textLogin_02()[pri:0, instance:TestCases.TC_LoginPage@741a8937]" started-at="2022-01-05T14:11:37 IST" name="textLogin_02" finished-at="2022-01-05T14:11:39 IST" duration-ms="1833" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at TestCases.TC_LoginPage.textLogin_02(TC_LoginPage.java:100)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- textLogin_02 -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:TestCases.TC_LoginPage@741a8937]" started-at="2022-01-05T14:11:39 IST" name="afterMethod" finished-at="2022-01-05T14:11:39 IST" duration-ms="231" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- TestCases.TC_LoginPage -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
